sbin/shipper_utils/update_and_install_workers.py

```diff
@@ -59,7 +60,15 @@ def update_docker_images(user, host, worker, autograding_workers, autograding_co
             except Exception as e:
               print(f"ERROR: Could not pull {image}")
               traceback.print_exc()
-              success = False
+
+              # check for machine
+              if platform.machine() == "aarch64":
+                  # TEMPORARY: docker pull often fails on ARM installation
+                  print("WARNING: SKIPPING DOCKER PULL ERROR")
```

Comment:
Could we include more specifics on what that error is. Given that hitting this requires a specific architecture not everyone has access to, more details would be appreciated just so might know if this were to get fixed at the engine level or something.